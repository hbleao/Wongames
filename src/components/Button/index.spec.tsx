import React from 'react';
import { screen, render } from '@testing-library/react';

import { Button } from '.';

const makeSut = () => {
  const sut = render(<Button />);

  return {
    sut
  };
};

describe('Button', () => {
  it('should be render a button', () => {
    makeSut();

    const button = screen.getByRole('button');

    expect(button).toBeInTheDocument();
  });
});
