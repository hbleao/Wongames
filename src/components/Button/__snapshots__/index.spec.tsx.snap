// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render the medium by size 1`] = `
<span>
  buy now
</span>
`;
